cardano-api/src/Cardano/Api/Certificate.hs

```diff
@@ -105,6 +109,52 @@ instance ToCBOR Certificate where
 instance FromCBOR Certificate where
     fromCBOR = fromShelleyCertificate <$> fromCBOR
 
+instance ToJSON Certificate where
```

Comment:
I'm wondering if it would be possible here to change constructors to records and then use [genericToJSON](https://hackage.haskell.org/package/aeson-2.1.2.1/docs/Data-Aeson.html#v:genericToJSON) with `constructorTagModifier`  instead of manual instances?

Reply:
I think we prefer explicit instances.  @Jimbo4350 do you have an opinion on `genericToJSON`?